MDI_Library/__init__.py

```diff
@@ -21,4 +21,4 @@
     MDI_Register_node, MDI_Check_node_exists, MDI_Get_nnodes, MDI_Get_node, \
     MDI_Register_command, MDI_Check_command_exists, MDI_Get_ncommands, MDI_Get_command, \
     MDI_Register_callback, MDI_Check_callback_exists, MDI_Get_ncallbacks, MDI_Get_callback, \
-    MDI_Plugin_get_argc, MDI_Plugin_get_arg, MDI_Launch_plugin
+    MDI_Plugin_get_argc, MDI_Plugin_get_arg, MDI_Launch_plugin, MDI_String_to_Atomic_Number
```

Comment:
I've been trying to switch the capitalization of MDI functions to conform to the MPI capitalization style, which only capitalizes the first letter after `MPI_`.  This should be `MDI_String_to_atomic_number`.  There are still some deprecated MDI functions that don't conform to this standard, but they generally just wrap MDI functions with the correct capitalization style.